TypeCobol/Compiler/CodeElements/Expressions/Expression.cs

```diff
@@ -121,13 +151,27 @@ public class ClassCondition : ConditionalExpression
 
 		public SyntaxProperty<bool> InvertResult { get; private set; }
 
+        protected override bool IsEquivalentOperator(Expression other)
+        {
+            var otherClassCondition = (ClassCondition) other;
+            return DataItem.SymbolReference?.Name == otherClassCondition.DataItem.SymbolReference?.Name
+                   && CharacterClassNameReference.Name == otherClassCondition.CharacterClassNameReference.Name
+                   && DataItemContentType.Value == otherClassCondition.DataItemContentType.Value
+                   && InvertResult.Value == otherClassCondition.InvertResult.Value;
```

Comment:
Making the remark here but concerns every `IsEquivalentOperator` implementations: some null checks seems to be missing. For example why add a null check on `InvertResult` in `ToString` and omit it here ?